hoard/models.py

```diff
@@ -74,6 +75,46 @@ def asdict(self) -> dict:
 
 
 # Helper functions
+def create_from_dublin_core_xml(data: str) -> Dataset:
+    record = ET.fromstring(data)
+    namespace = {
+        "oai": "http://www.openarchives.org/OAI/2.0/",
+        "dc": "http://purl.org/dc/elements/1.1/",
+    }
+    title_elem = record.find(".//dc:title", namespace)
+    if title_elem is not None and title_elem.text is not None:
+        title = title_elem.text
+    else:
+        raise Exception("No title field")
+    for x in record.findall(".//dc:creator", namespace):
```

Comment:
This will replace the `authors` variable every time you cycle through the `for x in record.findall...` loop, rather than appending each author to the list. Because this is the kind of thing that more comprehensive testing would catch, I would suggest using some actual OAI XML records from the WHOAS repository for testing, and test for every single Dataset field we are creating. Ideally you would include records that have multiple values for fields that are repeatable in DSpace (like creator) as well as no values for fields that are optional in DSpace (everything but title and publication date).

Reply:
Good catch, this was working properly but I didn't think of this issue in my last revision. Should I essentially be creating the WHOAS version of the `jpal_oai_server` fixture? And should that be in this PR or another?

Reply:
Yeah I think that's a good idea. I would go ahead and do it in this PR since you'll need to update the code and the tests anyway.

Reply:
Will do!